packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -277,6 +275,15 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     }
   };
 
+  private _calculatePrecision = (props: ISpinButtonProps) => {
+    if (props.step) {
+      const { precision = Math.max(calculatePrecision(props.step!), 0) } = props;
```

Comment:
`!` no longer necessary

Reply:
Actually best to fix the typings

Add something like 
```ts
public props: ISpinButtonProps & Required<Pick<ISpinButtonProps, 'step' | 'min'>>
```

into the class declaration and it should allow you to remove all usages of `!` operator

Reply:
```ts
type DefaultProps = Required<Pick<ISpinButtonProps, 'step' | 'min' | ... >>;
static defaultProps: DefaultProps =  {
    step: 1,
    min: 0,
    max: 100,
    disabled: false,
    labelPosition: Position.start,
    label: '',
    incrementButtonIcon: { iconName: 'ChevronUpSmall' },
    decrementButtonIcon: { iconName: 'ChevronDownSmall' }
  };

  public props: ISpinButtonProps & DefaultProps;
```